sdv/__init__.py

```diff
@@ -7,3 +7,23 @@
 __author__ = 'DataCebo, Inc.'
 __email__ = 'info@sdv.dev'
 __version__ = '1.0.1.dev0'
+
+import warnings
+
+from pkg_resources import iter_entry_points
+
+
+def _add_version():
+    for entry_point in iter_entry_points(name='version', group='sdv_modules'):
```

Comment:
I'm wondering if we can make this method a general purpose method that finds all of the new modules to be added in the group `sdv_modules` and then just adds them to the globals based on the `__name__`. What do you think of that?

Reply:
I think that might cause issues if we want to have add-ons for specific submodules instead of at the top level like we do here. Unless we want to have submodule add-ons have a different group?

Reply:
That's exactly what I was thinking. As a way of limiting the number of functions we have to write here, we can have one that adds modules directly at the current level and it knows that it's safe because they're in the `sdv_modules` group. If we wanted to add a new module to `sdv.metadata`, then we should have another function like this in the `sdv.metadata.__init__.py` that looks for the group `sdv.metadata_modules` or something like that. I just think that adding a full module to a package is something that might happen frequently. I'm open to ideas on this though

Reply:
I think that makes sense, I made the changes. If we want to be able to load add-ons for a submodule, we just need to add `sdv._addons._find_addons` to the `__init__.py` with whatever group name we decide to use for that submodule.

Reply:
If we like this format more, I'll update the other libraries as well